modules/image-loading-optimization/module.php

```diff
@@ -0,0 +1,299 @@
+<?php
+namespace Elementor\Modules\ImageLoadingOptimization;
+
+use Elementor\Core\Base\Module as BaseModule;
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly
+}
+
+class Module extends BaseModule {
+
+	private $min_priority_img_pixels;
+	private static $image_visited = [];
+
+	public function get_name() {
+		return 'image-loading-optimization';
+	}
+
+	public function __construct() {
+		parent::__construct();
+
+		$this->min_priority_img_pixels = 50000;
+
+		// Stop wp core logic.
+		add_action( 'init', [ $this, 'stop_core_fetchpriority_high_logic' ] );
+		add_filter( 'wp_lazy_loading_enabled', '__return_false' );
+
+		// Run optimization logic on header.
+		add_action( 'get_header', [ $this, 'set_buffer' ] );
+		add_action( 'elementor/page_templates/header-footer/before_content', [ $this, 'flush_buffer' ] );
+
+		// Run optimization logic on content.
+		add_filter( 'wp_get_attachment_image_attributes', [ $this, 'remove_get_attachment_loading_attributes' ], 10, 3 );
```

Comment:
I'm curious why this is needed, since we're disabling these attributes in lines 25 and 26 above? Also, nitpick: since the callback is only using the first parameter, this could be simplified to:
```suggestion
		add_filter( 'wp_get_attachment_image_attributes', [ $this, 'remove_get_attachment_loading_attributes' ] );
```